hms-app/src/components/IdeaCardFoldable.tsx

```diff
@@ -16,7 +16,7 @@ import { Idea, IdeaPreview } from '../common/types'
 import { getIdeaDetails } from '../actions/GetBackendData'
 
 type IProps = {
-  ideaPreview: IdeaPreview
+  ideas: Idea
```

Comment:
Since it is just singular
```suggestion
  idea: Idea
```